lib/types/api.types.ts

```diff
@@ -143,6 +143,8 @@ export interface AccessAPI {
   canEditAppConfig: () => Promise<boolean>
 }
 
+type EntityScopedIds = 'field' | 'entry' | 'contentType'
```

Comment:
Why not `EntryScopedIds`? We do not have an app location for any other entity.

Reply:
Fair point